### ISSUE TITLE
Usunięcie buttona

### ISSUE THREAD

**Issue by ErykMalinowski** [daftcode/daftacademy-frontend4beginners-autumn2019#19]
Czy w tym punkcie pracy domowej dopuszczalne jest użycie funkcji replaceWith czy jednak można przez to stracić punkty?

E: Jakby kogoś interesowało, to replaceWith jest ok.

**Comment by scieplak** [comment]
Tak, `replaceWith` jest jak najbardziej w porządku - ważne, żeby element został podmieniony.
Otworzę na razie issue ponownie, może jeszcze komuś się przyda ta informacja.
